prow/plugins/config.go

```diff
@@ -846,9 +841,9 @@ func (c *Configuration) EnabledReposForExternalPlugin(plugin string) (orgs, repo
 }
 
 // SetDefaults sets default options for config updating
-func (c *ConfigUpdater) SetDefaults() {
```

Comment:
whats wrong with this?

Reply:
Because this receiver is `cu` in other functions.